tests/testcontainers-env/src/main/java/org/alfresco/tas/DatabaseType.java

```diff
@@ -34,6 +53,28 @@ public String getUrl()
         return url;
     }
 
+    public String getSpringDatasourceUrl()
+    {
+        if(this.equals(DatabaseType.MSSQL_DB))
+        {
+            return getUrl().replace("\\", "");
```

Comment:
you first use these values, perhaps to escape a character, then you replace them with nothing? Please explain

Reply:
Url passed to container definition as JAVA_OPTS needs to have ';' escaped, therefore '\\' there. Url passed as SPRING_DATASOURCE_URL doesn't need that and is incorrect when left with '\\'.

Reply:
changed according to suggestions